mentat/session_input.py

```diff
@@ -20,6 +23,20 @@ async def collect_user_input(**kwargs: Any) -> StreamMessage:
 
     logging.debug(f"User Input: {response.data}")
 
+    # Intercept and run commands
+    if not kwargs.get("plain"):
```

Comment:
Instead of supporting and `**kwargs` for ` collect_user_input`, I think we should add a specific flag argument (e.g. `disable_commands: bool`) for disabling command interception since it's a lot clearer what's going on.

Reply:
Just changed how things work a tiny bit; I still have plain disabling commands, but it isn't in kwargs anymore. I think we should keep plain and disable_commands as one thing though, since the entire point of the plain is that it doesn't have autocomplete (like commands)

Reply:
Sg, I didn't understand what "plain" meant until now ("a 'plain' prompt")